src/components/data_display/helpers.ts

```diff
@@ -202,13 +203,15 @@ const getSortColumn = (
 const buildColumnList = (
   dataView: DataView,
   allColumns: string[],
-  type: 'TABLE' | 'DETAIL' | 'FILTER'
+  type: 'TABLE' | 'DETAIL' | 'FILTER' | 'RAW'
 ): string[] => {
   const includeField =
     type === 'TABLE'
       ? 'tableViewIncludeColumns'
       : type === 'DETAIL'
       ? 'detailViewIncludeColumns'
+      : type === 'RAW'
+      ? `rawDataIncludeColumns` ?? `tableViewIncludeColumns`
```

Comment:
This sounds more like a switch statement ? Not really sure how to read nested ternary

Reply:
People often complain about nested ternaries, but I generally find that the way VSCode formats them makes them fairly straightforward to read. It's basically a long chain of fallbacks -- keep going down until you hit a line that returns `true`, then return whatever is on the line immediately below. 

Yeah, could be formatted as a `switch`, but I'm not a huge fan of `switch (true)`, which is what this would be. (Plus was trying to make as small a change as I could here, since the logic gets a bit gnarly.)